packages/sync/src/offline/OfflineQueue.ts

```diff
@@ -50,7 +51,7 @@ export class OfflineQueue extends OperationQueue {
   }
 
   protected enqueueEntry(entry: OperationQueueEntry) {
-    if (this.squashOperations) {
+    if (this.squashOperations && !isNotSquashable(entry.operation)) {
```

Comment:
We have `NotIsNotSquashable` to decode that to real life:
- Squashable (hey we squash things)
- Not (We do not)
- Not Again (So apparently we do)

This is the only usage of this method so maybe we could avoid double negation?
How about having `isSquashable` :)